examples/go/hello-world/README.md

```diff
@@ -0,0 +1,83 @@
+# Hello, world!
+
+As customary, "hello, world!" is the simplest first app one would write and extend on a new platform.
+This app:
+
+- Provides a [manifest](https://github.com/mattermost/mattermost-plugin-apps/blob/master/examples/go/hello-world/hello.go#:~:text=var%20Manifest)
+  ([docs](https://developers.mattermost.com/integrate/apps/api/manifest/)), in
+  which it declares itself an HTTP app that acts as a bot, and binds to
+  locations in the user interface.
+- Contains a [send form](https://github.com/mattermost/mattermost-plugin-apps/blob/master/examples/go/hello-world/hello.go#:~:text=var%20SendForm)
+  ([docs](https://developers.mattermost.com/integrate/apps/api/interactivity/))
+  that allows to enter a custom message.
+- [Binds](https://github.com/mattermost/mattermost-plugin-apps/blob/master/examples/go/hello-world/hello.go#:~:text=Bindinings%20callback)
+  ([docs](https://developers.mattermost.com/integrate/apps/api/bindings/)) the
+  form to the channel header, and defines a `/helloworld send` command.
+- Contains the [Send](https://github.com/mattermost/mattermost-plugin-apps/blob/master/examples/go/hello-world/hello.go#:~:text=func%20Send)
+  call ([docs](https://developers.mattermost.com/integrate/apps/api/call/)) handler that sends a parameterized message back to the user.
+
+## Prerequisites
+
+Before you can start with your app, you first need to set up a local developer
```

Comment:
In cases where a system integrator isn't keen on working with Mattermost source code directly, and doesn't want to have to clone both the server & webapp source code locally, is there a way for that system integrator to create and test new Mattermost Apps?

Reply:
Yes, there is a docker-based dev. environment, we need to consolidate and make the docs (and the tech) consistent.

cc @mickmister